data/items.lua

```diff
@@ -198,7 +221,7 @@ function items:AttachItemInfo(data)
     sellPrice = sellPrice,
     classID = classID,
     subclassID = subclassID,
-    bindType = bindType,
+    bindType = bindType --[[@as Enum.ItemBind]],
```

Comment:
GetItemInfo wiki and WoW API use out of date number or LE_ITEM_BIND instead of Enum.ItemBind
Maybe still true for classic?